COMP120_Evaluation.tex

```diff
@@ -0,0 +1,45 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Evaluation}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Evaluation}
+
+% Please put your student number in the author field
+\author{James (JH182233) 1506530}
+
+\begin{document}
+
+\maketitle
+
+\section{Time Keeping}
+
+Over the course I found that I kept getting distracted from my projects. For me the most significant issue was not practising the practical side of programming, at first I felt lost, this might be due to the fact that i had been out of education for an extended period of time. Unlike being employed, education requires you to demonstrate your own timekeeping more strictly than employment would. Having experienced university I now realise I have not sufficiently developed my ability to use my free time efficiently. I will schedule my time and note it to keep track of my progress, by creating a daily timetable that will account for programming practice and leisure.
+
+\section{Communication}
+
+Previously in my old employment I know that being in charge meant that I had to communicate with the staff in order to work efficiently. In class I have seen a great lack of communication, for me its mainly my pride that is the cause. Initially I felt uncomfortable asking questions to anyone. As time went on though I became comfortable talking to the tutors about my issues. Having discussed my position in class with my tutor I now realise that I was not the only one. Having learnt this I have significantly improved my ability to communicate with my peers by taking charge of a few aspects of our group. For instance becoming a student rep. 
+
+\section{Essay}
+
+I found the essays stressful yet more engaging than the projects. For me the most important element for an essay is the research and referencing which is reflected in my marks. Equally the analytical and technical content is important for any essay to be successful. Subsequently I noticed that I had not done so well the these two fields, this reflects a lack of effort on certain aspects of my work. This may be due to a slight lack of interest in the academic side of the course or perhaps a lack of understanding.
```

Comment:
Requires more analysis into *why* it was not interesting and/or *what* affects your motivation. An action also needs to be proposed.